panic at github.com/anacrolix/torrent/cmd/torrentfs/main.go:72

After fixing #100 I tried putting http://old-releases.ubuntu.com/releases/14.04.2/ubuntu-14.04.2-desktop-amd64.iso.torrent into the torrents dir and starting torrentfs:

$ torrentfs -mountDir mnt -torrentPath tr
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x407354]

goroutine 19 [running]:
panic(0xae5200, 0xc8200100f0)
	/home/ctn/go/src/runtime/panic.go:481 +0x3e6
main.addTestPeer(0xc82010ce00)
	/home/ctn/src/github.com/anacrolix/torrent/cmd/torrentfs/main.go:72 +0x154
main.main.func3(0xc82010ce00)
	/home/ctn/src/github.com/anacrolix/torrent/cmd/torrentfs/main.go:142 +0x21
created by main.main
	/home/ctn/src/github.com/anacrolix/torrent/cmd/torrentfs/main.go:145 +0x68e

--- a/cmd/torrentfs/main.go
+++ b/cmd/torrentfs/main.go
@@ -137,12 +137,15 @@ func main() {
 	resolveTestPeerAddr()
 	fs := torrentfs.New(client)
 	go exitSignalHandlers(fs)
-	go func() {
-		for {
-			addTestPeer(client)
-			time.Sleep(10 * time.Second)
-		}
-	}()
+
+	if testPeerAddr != nil {
+		go func() {
+			for {
+				addTestPeer(client)
+				time.Sleep(10 * time.Second)
+			}
+		}()
+	}
 
 	if err := fusefs.Serve(conn, fs); err != nil {
 		log.Fatal(err)

